lib/tasks/fedramp.rake

```diff
@@ -0,0 +1,59 @@
+require 'io/console'
+
+namespace :rh_cloud do |args|
+  desc 'Register with FedRamp registration API. \
+        Run with INSTANCE=temp, stage, prod to specify an instance to connect to. \
+        Specify org_id=x replace your organization ID with x. \
+        Specify SATELLITE_RH_CLOUD_URL=https://x with the FedRamp endpoint you are connecting to.'
```

Comment:
Can this help text be revised? Like, if `INSTANCE` is set, why would `SATELLITE_RH_CLOUD_URL` need to be set, and vice versa?

Reply:
Good catch I will update it, instance is no longer needed.

Reply:
Updated the wording @Ichimonji10